odl/discr/discr_mappings.py

```diff
@@ -444,13 +425,14 @@ def nearest(arg, out=None):
 
     def __repr__(self):
         """Return ``repr(self)``."""
-        inner_str = '\n  {!r},\n  {!r},\n  {!r}'.format(self.range,
-                                                        self.grid,
-                                                        self.domain)
-        if self.order == 'F':
-            inner_str += ",\n  order='F'"
-        if self._variant == 'right':
-            inner_str += ",\n  variant='right'"
+        inner_str = '\n  {!r},\n  {!r},\n  {!r}'.format(
+            self.range, self.grid, self.domain)
+        sep = ',\n '
```

Comment:
this "sep" style has to be changable, right? Tbh all of these repr guys need to be changable, we could make a util for it. Something like:

```python
def repr_util(obj, members, defaults):
    """
    obj : object to print
    members : list of strings of members
    defaults : dict with defaults
    """
    formated_members = []
    for member in members:
        member_value = getattr(obj, member)
        if member in defaults and member_value == defaults[member]:
            continue
            
        formated_members += ['{!r}={!r}'.format(member, member_value)]
    
    inner_str = ', '.join(formated_members)
    
    return '{}({})'.format(obj.__class__.__name__, inner_str)
```